include/glow/Backends/CompiledFunction.h

```diff
@@ -16,8 +16,16 @@
 #ifndef GLOW_BACKENDS_COMPILEDFUNCTION_H
 #define GLOW_BACKENDS_COMPILEDFUNCTION_H
 
+#include <unordered_map>
+
 namespace glow {
 
+class Placeholder;
+class Tensor;
+
+/// Maps placeholders to the tensors that back them.
+using PlaceholderMap = std::unordered_map<Placeholder *, Tensor *>;
```

Comment:
@qcolombet @hegemanjwh2 @artemrakhov @rdzhabarov I used the type "PlaceholderMap" which is an alias to some concrete type. If we ever want to change it to something else (vector, llvm::densemap, etc) it would be really easy to do.